nixos/modules/services/web-apps/gotosocial.md

```diff
@@ -0,0 +1,64 @@
+# GoToSocial {#module-services-gotosocial}
+
+[GoToSocial](https://gotosocial.org/) is an ActivityPub social network server, written in Golang.
+
+## Service configuration {#modules-services-gotosocial-service-configuration}
+
+The following configuration sets up the PostgreSQL as database backend and binds
+GoToSocial to `127.0.0.1:31337`, expecting to be run behind a HTTP proxy on `gotosocial.example.com`.
```

Comment:
```suggestion
GoToSocial to `127.0.0.1:3137`, expecting to be run behind a HTTP proxy on `gotosocial.example.com`.
```
Using such a high port could conflict with other things maybe, we should stick to something below 10k

Reply:
Fixed!